File: Session_01_BasicPagramming/Lab01/Python_Lang/Demo06.py
#
#  Created by NDHuy on 05/04/2020
#  Copyright 2020 by SEstudio
#  Input one character. Change to Upper or Lower
#

def main():
    c = input("Input character c = ")
    
    # Get character from String
    c = c[0]

    # Convert character to ASCII code
    ascii_code = ord(c)

    if (ascii_code >= 65) and (ascii_code <= 90):
        ascii_code = ascii_code + 32
    elif (ascii_code >= 97) and (ascii_code <= 122):
            ascii_code = ascii_code - 32 
    
    # Convert ASCII code to character
    c = chr(ascii_code)

    print("Result of convert character c = %c" %(c))


if __name__ == "__main__":
    main()

File: 01_BasicPagramming/Lab01/C_Lang/src/Demo06.cpp
//
//  Created by NDHuy on 05/04/2020
//  Copyright 2020 by SEstudio
//  Input one character. Change to Upper or Lower
//

#include <stdio.h>

int main()
{
    char c;
    printf("Input character c = ");
    scanf("%c", &c);

    if((c >= 65) && (c <= 90))
    {
        c = c + 32;
    }
    else 
    {
        if ((c >= 97) && (c <= 122))
        {
            c -= 32;
        }
    }

    printf("Result of convert character c = %c \n", c);

    return 0;
}